src/components/Login/index.jsx

```diff
@@ -0,0 +1,165 @@
+    let { email, password } = loginCredentials;
+    let errors = {};
+    if (!validateEmail(email)) {
+      errors["email"] = "Email format error";
+    }
+    if (email.trim() === "") {
+      errors["email"] = "Please fill this field";
+    }
+    if (password.trim().length < 6) {
+      errors["password"] = "Password must be at least 6 characters long";
+    }
+    if (password.trim() === "") {
+      errors["password"] = "Please fill this field";
+    }
+    setErrors(errors);
+    if (Object.keys(errors).length > 0) {
+      return;
+    }
+    props.onSubmit(loginCredentials);
+  };
+  const handleFacebookAuth = () => {
+    props.onFacebookAuth();
+  };
+  const handleGoogleAuth = () => {
+    props.onGoogleAuth();
+  };
+  const handleGoToRegister = () => {
+    props.onGoToRegister();
+  };
+  const handleGoToPasswordReset = () => {
+    props.onGoToPasswordReset();
+  };
+
+  return (
+    <form noValidate onSubmit={handleSubmit}>
+      <div className="loginContainer">
+        <div className="loginContainer__loginTitle">{t("login.login")}</div>
+        {props.error && (
+          <Alert
+            style={{ marginBottom: 10 }}
+            type="error"
+            showIcon
+            message={props.error}
+          ></Alert>
+        )}
+        {props.message}
+        <div className="loginContainer__loginDialog">
+          <div className="loginContainer__loginDialog__inputLabel">
+            {t("login.email")}
+          </div>
+          <div className="loginContainer__loginDialog__input">
+            <Input
+              className={
+                errors["email"] &&
+                "loginContainer__loginDialog__input__hasError"
+              }
+              type="email"
+              value={loginCredentials.email}
+              onChange={(e) => {
+                handleChange("email", e.target.value);
+              }}
+            />
+            {errors["email"] && (
+              <div className="loginContainer__loginDialog__errorContainer">
+                {errors["email"]}
+              </div>
+            )}
+          </div>
+          <div className="loginContainer__loginDialog__inputLabel">
+            {t("login.password")}
+          </div>
+          <div className="loginContainer__loginDialog__input">
+            <Input
+              className={
+                errors["password"] &&
+                "loginContainer__loginDialog__input__hasError"
+              }
+              type="password"
+              value={loginCredentials.password}
+              onChange={(e) => {
+                handleChange("password", e.target.value);
+              }}
+            />
+            {errors["password"] && (
+              <div className="loginContainer__loginDialog__errorContainer">
+                {errors["password"]}
+              </div>
+            )}
+          </div>
+          <div className="loginContainer__loginDialog__forgotPasswordContainer">
+            <a
+              href="/#"
```

Comment:
if you do # instead of /# you don't need to prevent default I think

Reply:
PTAL